### PR TITLE
Update `BAT`

### PR DESCRIPTION
**Pull request by ghost**
Added -Ady on the team, fixed spelling, removed some extra spaces, added note, fixed wording etc.

### CODE REVIEW

**Review by Lekuruu — DISMISSED** [review]
yep

File: wiki/People/BAT/en.md [hunk 1]
```diff
@@ -82,5 +83,6 @@ User | Time in BAT
 [^3]: https://discord.com/channels/1152925764262580236/1152925765566988341/1242819755241635880
 [^4]: https://discord.com/channels/1152925764262580236/1152925765566988341/1251193374438854706
 [^5]: https://osu.titanic.sh/forum/20/t/472/
+[^6]: -Ady joined the BAT in May 19th 2024, resigned in August 13th 2024, joined BAT in October 22nd 2024.
```
Comment (BlueChinchompa) [inline wiki/People/BAT/en.md:86]:
```suggestion
[^6]: -Ady joined the BAT in May 19th 2024, resigned in August 13th 2024, rejoined BAT on October 22nd 2024.
```

**Review by BlueChinchompa — APPROVED** [review]
Looks good!

**Review by Meruru-chan — APPROVED** [review]
Ok

**Pull request merged**